tools/webcomponent/REAME.md

```diff
@@ -0,0 +1,23 @@
+# Web components tools
+
+## HTML embedder
+
+The file [`wc-embedder.html`](wc-embedder.html) can be used to wrap a geonetwork-ui Web Component into a full HTML page,
+for example to be used in an [iframe](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe).
+
+To use it, specify the name and attributes of the Web Component to be created when accessing the page:
+
+```
+wc-embedder.html?e=gn-dataset-preview&a=api-url=https://dev.geo2france.fr/geonetwork/srv/api&a=primary-color=%230f4395&a=secondary-color=%238bc832&a=main-color=%23555&a=background-color=%23fdfbff
```

Comment:
we will need a kind of api doc (up to date) for the web components...

Reply:
Sure, and all the things related to it. Hopefully we can have a proper technical documentation website soon